src/content/en/tools/chrome-devtools/remote-debugging/index.md

```diff
@@ -2,151 +2,152 @@ project_path: /web/_project.yaml
 
-From here you can see the status of all connected remote devices. You don't
-have any devices connected right now, so your dialog should look similar to the
-following screenshot. Make sure that **Discover USB devices** is enabled. 
+1. In DevTools, click the **Settings** tab, if another tab is showing.
 
-![empty inspect devices dialog](imgs/empty-inspect-devices.png)
+1. Make sure that **Discover USB devices** is enabled.
 
-Connect your Android device to your development machine using a USB cable. You 
-should connect your Android device directly to your development machine, not
-through any intermediate hubs. 
+     ![Discover USB devices is enabled][discover]
 
-If this is your first time connecting this Android device to this development
-machine, you should see an unknown device that is pending authorization on
-your Inspect Devices dialog.
+[discover]: /web/tools/chrome-devtools/remote-debugging/imgs/discover-usb-devices.png
 
-![pending authorization](imgs/pending-authorization.png)
+1. Connect your Android device directly to your development machine using a USB
+   cable. Don't use any intermediate USB hubs. If this is your first time
+   connecting your Android device to this development machine, your device
+   shows up under **Unknown**, with the text **Pending Authorization** below
+   it.
 
-If this is the case, then you need to open the **Allow USB debugging** prompt
-on your Android device and grant the permissions. 
+       ![Unknown device, pending authorization][unknown]
 
-**Tip**: If you have encounter any issues during the discovery process, you 
-can start the process anew by going back to **Developer options** and tapping 
-**Revoke USB debugging authorizations**. 
+[unknown]: /web/tools/chrome-devtools/remote-debugging/imgs/unknown-device.png
 
-After allowing USB debugging, you should see your device in the Inspect Devices
-dialog. 
+1. If your device is showing up as **Unknown**, accept the **Allow USB
+   Debugging** permission prompt on your Android device. **Unknown** is
+   replaced with the model name of your Android device. The green circle
+   and the **Connected** text indicate that you are all set to remotely
+   debug your Android device from your development machine.
 
-![inspect devices dialog with connection](imgs/connected-inspect-devices.png)
+Note: If you have any issues during the discovery process, you 
+can restart the process by selecting **Settings** > **Developer Options** >
+**Revoke USB Debugging Authorizations** on your Android device.
 
-## Debug content on Android device from development machine
+## Step 2: Debug content on Android device from development machine {#debug}
 
-From the Inspect Devices dialog, select your device from the menu on the left.
+1. If you don't already have Chrome open on your Android device, open it now.
 
-![inspecting a device](imgs/selected-inspect-device.png)
+1. Back in DevTools, click on the tab that matches your device's
```

Comment:
'Click the tab', not 'click on the tab'. This is a Google rule. Also Strunk and White. Do throughout.